helmfile/charts/user-rbac/templates/rolebindings/workload-admin.yaml

```diff
@@ -29,6 +29,9 @@ subjects:
 {{- $value := lookup "rbac.authorization.k8s.io/v1" "RoleBinding" $namespace.name "extra-workload-admins" }}
 {{- if and $known (not $value) (or $.Release.IsInstall $.Release.IsUpgrade) }}
 ---
+# NOTE: This will always be deleted from the Helm release on the second apply
+# but the role binding won't be deleted due to the
+# helm.sh/resource-policy: keep annotation.
```

Comment:
This text looks good. But could you also add a line about why we are doing this in the first place, i.e. because this is something that the app devs should update and we do not want to overwrite their changes.

Reply:
What about this wording? https://github.com/elastisys/compliantkubernetes-apps/pull/1785/commits/104e32955ae317457040323019759c08cb2c453d